File: extensions/resteasy-reactive/rest-client-reactive/deployment/src/test/java/io/quarkus/rest/client/reactive/stork/HelloClient.java
package io.quarkus.rest.client.reactive.stork;

import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;

import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;

@Path("/")
@RegisterRestClient(configKey = "hello2")
public interface HelloClient {
    @GET
    String hello();
}

File: integration-tests/resteasy-reactive-kotlin/standard/src/main/kotlin/io/quarkus/it/resteasy/reactive/kotlin/ft/HelloClient.kt
package io.quarkus.it.resteasy.reactive.kotlin.ft

import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import org.eclipse.microprofile.faulttolerance.Fallback
import org.eclipse.microprofile.rest.client.inject.RegisterRestClient

@Path("/ft/hello")
@RegisterRestClient(configKey = "ft-hello")
interface HelloClient {
    @GET
    @Fallback(fallbackMethod = "fallback")
    suspend fun hello(): String

    suspend fun fallback() = "fallback"
}
